src/main/java/RangeableRandomNumber.java

```diff
@@ -0,0 +1,30 @@
+public class RangeableRandomNumber extends AbstractRandomNumber {
+
+    private static final int DEFAULT_MAX_RANDOM_NUMBER = RangeableRandomNumberFactory.DEFAULT_MAX_RANDOM_NUMBER;
+    private static final int DEFAULT_MIN_RANDOM_NUMBER = RangeableRandomNumberFactory.DEFAULT_MIN_RANDOM_NUMBER;
+
+    private final int max;
+
+    private final int min;
+
+    public RangeableRandomNumber(int number, int max, int min) {
+        super(number);
+        validateNumber(number);
+        this.max = max;
+        this.min = min;
+    }
+
+    public RangeableRandomNumber(int number) {
+        super(number);
+        this.max = DEFAULT_MAX_RANDOM_NUMBER;
+        this.min = DEFAULT_MIN_RANDOM_NUMBER;
+        validateNumber(number);
+    }
+
```

Comment:
public RangeableRandomNumber(int number) {
        this(number, DEFAULT_MAX_RANDOM_NUMBER, DEFAULT_MIN_RANDOM_NUMBER);
    }

😇